CMakeLists.txt

```diff
@@ -112,8 +117,10 @@ if (INSTALL)
       set(LIBRARY_BASE ${INSTALL})
       # This is a terrible hack, but hey it works
       set(CMAKE_BUILD_RPATH "$ORIGIN:$ORIGIN/..")
-else()
+elseif(WRAP_PYTHON)
       set(MODULE_BASE ${PROJECT_BINARY_DIR}/neml)
+      set(LIBRARY_BASE ${PROJECT_BINARY_DIR}/neml)
+else()
       set(LIBRARY_BASE ${PROJECT_BINARY_DIR}/lib)
 endif()
```

Comment:
Compared to the previous version we are not setting `MODULE_BASE` for the `else` case

Reply:
Yes, we're doing things a little different for windows now (emitting the libneml into the neml directory for the python bindings).